### ISSUE TITLE
`HISourcesSmallCatalog` returns SkyModel instead of path

### ISSUE THREAD

**Issue by sfiruch** [i4Ds/Karabo-Pipeline#605]
When one wants to use the "HISourcesSmallCatalog", you have to parse the results manually, as seen in https://github.com/i4Ds/Karabo-Pipeline/blob/main/karabo/examples/LineEmissionBackendsComparison.ipynb:
```
catalog_path = HISourcesSmallCatalogDownloadObject().get()
print(catalog_path)
sky = SkyModel.get_sky_model_from_h5_to_xarray(
    path=catalog_path,
)
```

Instead of a path, the function should return the parsed results directly, i.e. a `SkyModel`.

**Comment by Lukas113** [comment]
I agree. Actually, I was also thinking of reducing the number of functions from the `SkyModel.get_*` area (like `get_MALS_DR1V3_Sky`, `get_MIGHTEE_Sky`, `get_GLEAM_Sky`, etc... to get it from:
- Litererals
- a different module which doesn't extend the `SkyModel` for each Sky which is added.
